linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/tests/correct.pass.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# packages = libpam-pkcs11
```

Comment:
I think you should add `# platform = multi_platform_ubuntu` to these test scenarios because of the package libpam-pkcs11. The rule smartcard_configure_cert_checking is applicable also on RHEL 7 and other systems and when you will try to run these test scenarios on a RHEL 7 machine it will fail because it can't install this package because it has a different name.

Reply:
fixed, thanks!